src/client/components/App.js

```diff
@@ -77,15 +77,15 @@ const activeStyle = {color:' #ffffff', background: '#006677'}
 const App = () => (
   <div>
     <ul className="nav nav-tabs">
-      <li><NavLink exact activeStyle={activeStyle} to='/'>Supplier</NavLink></li>
-      <li><NavLink activeStyle={activeStyle} to='/registration'>Supplier Registration</NavLink></li>
-      <li><NavLink activeStyle={activeStyle} to='/address'>Supplier Address</NavLink></li>
-      <li><NavLink activeStyle={activeStyle} to='/contact'>Supplier Contact</NavLink></li>
+      <li><NavLink exact activeStyle={activeStyle} to='/supplier'>Supplier</NavLink></li>
```

Comment:
instead of hardcoding service name in routes you can use history to do that automatically, like here:
https://github.com/OpusCapita/onboarding/blob/develop/src/client/containers/CampaignsApplication.react.js#L35